### PR TITLE
Helios endpoint improvements

### PR DESCRIPTION
**Pull request by ArturKlajnerok**
Please take a look: @michalroszka

### CODE REVIEW

File: extensions/wikia/Helios/HelperController.class.php [hunk 1]
```diff
@@ -15,8 +15,10 @@ public function checkAntiSpoof()
 	{
 		$this->response->setFormat( 'json' );
 		$this->response->setCacheValidity( \WikiaResponse::CACHE_DISABLED );
+		$this->response->setVal( 'allow', false );
```
Comment (michalroszka) [inline extensions/wikia/Helios/HelperController.class.php]: @ArturKlajnerok — this one is not needed here as it is set in `return` statements.

File: extensions/wikia/Helios/HelperController.class.php [hunk 2]
```diff
@@ -52,7 +56,7 @@ public function updateAntiSpoof()
 
 		if ( !empty( $this->wg->EnableAntiSpoofExt ) ) {
 			$oSpoofUser = new \SpoofUser( $sName );
-			$oSpoofUser->record();
+			return $this->response->setVal( 'success', $oSpoofUser->record() );
```
Comment (michalroszka) [inline extensions/wikia/Helios/HelperController.class.php]: `SpoofUser::record()` returns true even on failure, so the check for success might be inconclusive and misleading.

File: extensions/wikia/Helios/HelperController.class.php [hunk 3]
```diff
@@ -95,7 +99,8 @@ public function sendConfirmationEmail()
 			return;
 		}
 
-		$sMemKey = $this->getMemKeyConfirmationEmailsSent( $oUser->getId() );
+		$userLoginHelper = ( new \UserLoginHelper );
```
Comment (michalroszka) [inline extensions/wikia/Helios/HelperController.class.php]: I would use the Hungarian notation for consistency, so `$oUserLoginHelper`.

File: extensions/wikia/Helios/HelperController.class.php (hunk 2 above)
Comment (ArturKlajnerok) [inline extensions/wikia/Helios/HelperController.class.php]: My plan is to return the result here to know what was the operation outcome and to fix SpoofUser::record() implementation.

**Pull request merged**